### PR TITLE
docs(rendering-elements): wording

### PR DESCRIPTION
**Pull request by KennethKinLum**

### CODE REVIEW

**Comment by chloewlin** [comment]
@KennethKinLum thank you for your feedback. We understand that it is more common to use props in the plural form in English. Our react community actually had several discussions around singular/plural nouns and decided to remove all suffixes (including -s and -es) for clarity and readability. You can find the guideline here: 

https://github.com/reactjs/zh-hant.reactjs.org/wiki/Glossary

File: content/docs/rendering-elements.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ const element = <h1>Hello, world</h1>;
 
 >**注意：**
 >
->大家可能會將 element 與更廣為人知的「component」概念混淆。我們將會在[下一個章節](/docs/components-and-props.html)介紹 component。Element 是由 component 所「組成」的，我們建議你在開始之前閱讀本章節。
+>大家可能會將 element 與更廣為人知的「component」概念混淆。我們將會在[下一個章節](/docs/components-and-props.html)介紹 component。Components 是由 elements 所「組成」的，我們建議你在開始之前閱讀本章節。
```
Comment (chloewlin) [inline content/docs/rendering-elements.md]:
```suggestion
>大家可能會將 element 與更廣為人知的「component」概念混淆。我們將會在[下一個章節](/docs/components-and-props.html)介紹 component。Component 是由 element 所「組成」的，我們建議你在開始之前閱讀本章節。
```

**Comment by KennethKinLum** [comment]
the singular rule can make sense in some situation. In this situation, if you don't use plural, it is very strange.  You'd say 

A component is made up of elements.
A house is made up of bricks.

It'd be very strange to say even in Chinese

A house is made up of brick.

In Chinese,
一個房子是由磚頭組成
is ok.  But
一個 house 是由 brick 組成
may sound weird.  But I guess it sound ok if the reader immediately translates it in their mind to "一個房子是由磚頭組成".

File: content/docs/rendering-elements.md (hunk 1 above)
Comment (KennethKinLum) [reply content/docs/rendering-elements.md]: @chloewlin the singular rule can make sense in some situation. In this situation, if you don't use plural, it is very strange.  You'd say 

A component is made up of elements.
A house is made up of bricks.

It'd be very strange to say even in Chinese

A house is made up of brick.

In Chinese,
一個房子是由磚頭組成
is ok.  But
一個 house 是由 brick 組成
may sound weird.  But I guess it sound ok if the reader immediately translates it in their mind to "一個房子是由磚頭組成".

**Comment by KennethKinLum** [comment]
> @KennethKinLum thank you for your feedback. We understand that it is more common to use props in the plural form in English. Our react community actually had several discussions around singular/plural nouns and decided to remove all suffixes (including -s and -es) for clarity and readability. You can find the guideline here: https://github.com/reactjs/zh-hant.reactjs.org/wiki/Glossary
> 
> I checked the original doc and this part has a translation error. But we would like to keep every word singular. Can you update your PR?

so I confirmed the change to make them singular... but the PR is still showing as "Change requested".  How do I confirm the change?

**Comment by neighborhood999** [comment]
Hi @KennethKinLum, can you update commit messages & PR title to English? Thanks

**Comment by KennethKinLum** [comment]
The original text: Elements are what components are “made of”

Now the translation means: Components are what elements are “made of”

**Comment by neighborhood999** [comment]
> The original text: Elements are what components are “made of”
> 
> Now the translation means: Components are what elements are “made of”
> 
> @chloewlin @neighborhood999

可以簡單一些，不需要寫這麼長，例如 `docs(rendering-elements): wording`

**Comment by jianlin** [comment]
Do I need to do anything else?

**Pull request merged**